check_k8s.py

```diff
@@ -78,5 +93,14 @@ def main():
     sys.exit(output.state.value)
 
 
+def is_ignored_resource(line, expressions):
+    word = re.search(r"\S+:{1}", line)
+    for expression in expressions:
+        found = re.search(expression, word.group())
```

Comment:
Since this is a double for-loop with potentially lots of entries I think we should compile these regexes before hand.

Reply:
Ok will do